README.md

```diff
@@ -66,7 +66,9 @@ You need to **implement** in your Flutter iOS project a **Widget Extension** & d
 
   <img alt="enable live activities xcode" src="https://raw.githubusercontent.com/istornz/live_activities/main/images/tutorial/enable_live_activities.gif" width="700px" />
 
-  - Create App Group for both ```Runner``` & your ```Widget Extension```.
+  - Add the "App Group" capability for both ```Runner``` and your widget extension.
+    After you add the capability, check the checkmark next to the text field that contains an identifier of the form `group.example.myapp`.
+    This identifier will be used later and refered to as `YOUR_GROUP_ID`.
```

Comment:
This is visible in the gif, but for people who are mostly reading the text (like me) I think it's helpful to spell it out as things will be broken later if this step is missed.